### PR TITLE
Merge test -> beta

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Merge test -> beta

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 1]
```diff
@@ -4585,6 +4585,14 @@ label {
     color: var(--dark-blue01) !important;
 }
 
+.primary-color {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:4588]: Rohit why you added this, We have color.css file you can find colors there

File: apps/web-giddh/src/app/shared/header/components/account-update-new-details/account-update-new-details.component.ts [hunk 2]
```diff
@@ -1073,7 +1078,13 @@ export class AccountUpdateNewDetailsComponent implements OnInit, OnDestroy, OnCh
                     if (res.country.currency) {
                         this.selectedCountryCurrency = res.country.currency.code;
                         this.selectedAccountCallingCode = res.country.callingCode;
+                        if(selectedAcountCurrency) {
                         this.addAccountForm.get('currency').patchValue(selectedAcountCurrency);
+                        this.selectedCurrency = selectedAcountCurrency;
+                        } else {
+                        this.addAccountForm.get('currency').patchValue(this.selectedCountryCurrency);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/account-update-new-details/account-update-new-details.component.ts:1085]: format code

File: apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts [hunk 3]
```diff
@@ -72,13 +75,25 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
...
+    public groupWithParentLessDetailsList:any[] = [];
     @ViewChild('deleteGroupModal', {static: true}) public deleteGroupModal: ModalDirective;
     @ViewChild('moveToGroupDropDown', {static: true}) public moveToGroupDropDown: ShSelectComponent;
+    /** To check is groups belongs to debtor or creditors type  */
+    public isDebtorCreditorGroups: boolean = false;
+    /** To check discount box show/hide */
+    public showDiscount: boolean = false;
+    /** Stores list of discount */
+    public discountList: any[] = [];
+    public discountList$: Observable<IDiscountList[]>;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts:88]: add js doc

File: apps/web-giddh/src/app/actions/accounts.actions.ts [hunk 4]
```diff
@@ -651,6 +652,26 @@ export class AccountsAction {
                 };
             })));
 
+    @Effect()
+    public ApplyAccountDiscountsV2$: Observable<Action> = this.action$
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/actions/accounts.actions.ts:656]: Consider using createEffect in place of @Effect

File: apps/web-giddh/src/app/actions/accounts.actions.ts [hunk 5]
```diff
@@ -651,6 +652,26 @@ export class AccountsAction {
...
+        .pipe(ofType(AccountsAction.APPLY_ACCOUNT_DISCOUNTS_V2),
+            switchMap((action: CustomActions) => this._accountService.applyDiscounts(action.payload)),
+            map(response => {
+                return this.applyAccountDiscountResponseV2(response);
+            }));
+
+    @Effect()
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/actions/accounts.actions.ts:663]: Consider using createEffect in place of @Effect

File: apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.ts [hunk 6]
```diff
@@ -17,7 +20,13 @@ export class UploadFileComponent implements OnInit {
     public selectedType: string = '';
     public title: string;
 
-    constructor(private _toaster: ToasterService) {
+    /** Subject to unsubscribe all the listeners */
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+
+    constructor(
+        private _toaster: ToasterService,
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.ts:27]: Avoid using `_` for private members

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Review by rohit-walkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss:4588]: @shalinee-walkover there was existing one as you can see above of this css, I  change primary_clr to primary-color due PR review changes I can not find primary blue color that why have added this one

**Review by shalinee-walkover — APPROVED** [review]
Approve

**Pull request merged**